code/modules/language/pohlan.dm

```diff
@@ -0,0 +1,14 @@
+/datum/language/pohlan
+	name = "Poh'lan"
+	desc = "The language of the Grod people. It is synthesized using chitters and grumbles, and is completely incomprehensible."
```

Comment:
I would add "to outsiders".
No language can be completely incomprehensible, it wouldn't be a language then, just random noises